presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -407,6 +410,7 @@ private ConnectorTableMetadata getTableMetadata(SchemaTableName tableName)
             properties.put(STORAGE_FORMAT_PROPERTY, format);
         }
         catch (PrestoException ignored) {
+            LOG.warn("Storage Format Error: %s", ignored.getMessage());
```

Comment:
We avoid adding warnings like this to Presto, since it will flood the logs and is not useful.

Reply:
Sure